### ISSUE TITLE
 Automatically initialize the package

### ISSUE THREAD

**Issue by luizfernandonb** [arich55/simple-block-cursor#3]
Would it be possible for the package to start by itself instead of every time you start Atom, the user has to press `Ctrl+Alt+O`?

**Comment by WarrenMarshall** [comment]
Yes, please!  This would be super useful.  I wondered if I was missing a checkbox somewhere.  I love a good block cursor...

**Comment by arich55** [comment]
Hey, not a bad idea - I take some time to look into this
